circuits/ts/__tests__/CalculateTotal.test.ts

```diff
@@ -1,16 +1,16 @@
-import { expect } from "chai";
-import tester from "circom_tester";
+import { WitnessTester } from "circomkit";
 
-import path from "path";
-
-import { getSignal } from "./utils/utils";
+import { circomkit } from "./utils/utils";
```

Comment:
The name is confusing to me hhh (I just found it's the instance instead of the package). Could we rename it? (if it's unnecessary, just ignore it)

Reply:
changed to circomkitInstance to be clearer